.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
-lib/**/*
\ No newline at end of file
+lib/**/*
+.secrets
```

Comment:
Consider adding more specific patterns for secrets instead of a general `.secrets` entry. This might accidentally ignore important files. For example, you could use patterns like `*.key`, `*.pem`, or specific file names that contain secrets.